app/src/main/java/org/mozilla/tv/firefox/architecture/KillswitchLayout.kt

```diff
@@ -23,10 +23,10 @@ sealed class KillswitchLocales {
  *
  * This is done to prevent bugs similar to #2133. In that bug, a view was set to visibility
  * GONE when disabled, however its animation toggled that visibility back to VISIBLE.
- * [KillswitchLinearLayout] allows us to be sure that something will not be displayed if
+ * [KillswitchLayout] allows us to be sure that something will not be displayed if
  * requirements are not set by intercepting [setVisibility] calls.
  */
-class KillswitchLinearLayout : LinearLayout {
+class KillswitchLayout : FrameLayout {
```

Comment:
nit: I wonder if making this a ConstraintLayout would be more useful, though it is more prescriptive so maybe not.

Reply:
I think `FrameLayout` is a good approach because it has minimal overhead, and consumers can use whatever layout they want inside it.